### PR TITLE
EclipseLink example

### PR DESCRIPTION
**Pull request by slobodanl**
Relation to wildfly-swarm PR #139

Providing example that use new JPA via EclipseLink module.

### CODE REVIEW

File: jpa/jpa-eclipselink/pom.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  <dependencies>
+    <dependency>
+      <groupId>org.wildfly.swarm.examples</groupId>
+      <artifactId>examples-base</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.spec.javax.servlet</groupId>
+      <artifactId>jboss-servlet-api_3.1_spec</artifactId>
+      <version>1.0.0.Final</version>
+    </dependency>
```
Comment (kenfinnigan) [inline jpa/jpa-eclipselink/pom.xml]: Does jboss-servlet-api_3.1_spec need to be compile scoped?

Comment (slobodanl) [inline jpa/jpa-eclipselink/pom.xml]: Actually this patch is based on already existing example jpa/jpa-war with minor change to reflect the new module Eclipselink in action. Not still completely sure that this example is even needed, yet i think it has value for the community and people using JPA over Eclipselink.
As for the scope i would say it should be "provided" but still it is inherited from jpa/jpa-war.

Comment (kenfinnigan) [inline jpa/jpa-eclipselink/pom.xml]: I actually checked back and saw that too.

Should probably be changed for both as "provided" should be all it needs

**Comment by kenfinnigan** [comment]
ok to test

**Comment by kenfinnigan** [comment]
retest this please

**Comment by kenfinnigan** [comment]
Hmm, looks like the test can't find the named query.

Could you take a look?

Thanks

**Pull request merged**